.github/dependabot.yml

```diff
@@ -0,0 +1,26 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+
+version: 2
+updates:
+  - package-ecosystem: "nuget" # See documentation for possible values
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "weekly"
+  - package-ecosystem: "docker" # See documentation for possible values
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "weekly"
+  - package-ecosystem: "terraform" # See documentation for possible values
+    directory: "/infrastructure" # Location of package manifests
+    schedule:
+      interval: "weekly"
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    groups:
+      all-actions:
+        patterns: [ "*" ]
```

Comment:
Sorry for the newbie questions here! But to check my understanding... this grouping will create a single PR for multiple dependency vulnerabilities that may be found in our github actions?

Why does `github-actions` have a group while the others don't?

Reply:
yes @johncollinson2001 you understand correctly: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#groups

I'm not sure it would make much sense to group ALL dependency vulnerabilities. in some cases it does, such as with something that is more "utility", so to say. does that make sense to you?

Reply:
That's great - thank you!